### PR TITLE
SUB-1601 Roames trigger dag gui

### PR DESCRIPTION
**Pull request by draffelt**

### CODE REVIEW

File: airflow/www/templates/airflow/dag.html [hunk 1]
```diff
@@ -54,40 +54,24 @@ <h4 class="pull-right">
...
         Graph View</a></li>
       <li><a href="{{ url_for("airflow.tree", dag_id=dag.dag_id, num_runs=25, root=root) }}">
           <span class="glyphicon glyphicon-tree-deciduous" aria-hidden="true"></span>
         Tree View
       </a></li>
-      <li><a href="{{ url_for("airflow.duration", dag_id=dag.dag_id, days=30, root=root) }}">
-          <span class="glyphicon glyphicon-stats" aria-hidden="true"></span>
-          Task Duration
-      </a></li>
-      <li><a href="{{ url_for("airflow.tries", dag_id=dag.dag_id, days=30, root=root) }}">
```
Comment (PrabodhaSrimal) [inline airflow/www/templates/airflow/dag.html]: Task Duration and Task Tries may be useful for debugging some DAGs

File: airflow/www/templates/airflow/dag.html [hunk 2]
```diff
@@ -54,40 +54,24 @@ <h4 class="pull-right">
...
-        <a href="{{ url_for("airflow.landing_times", dag_id=dag.dag_id, days=30, root=root) }}">
-          <span class="glyphicon glyphicon-plane" aria-hidden="true"></span>
-          Landing Times
-        </a>
-      </li>
       <li>
         <a href="{{ url_for("airflow.gantt", dag_id=dag.dag_id, root=root) }}">
           <span class="glyphicon glyphicon-align-left" aria-hidden="true"></span>
           <i class="icon-align-left"></i>
           Gantt
```
Comment (PrabodhaSrimal) [inline airflow/www/templates/airflow/dag.html:88]: Gantt may be removed.

File: airflow/www/templates/airflow/dags.html [hunk 3]
```diff
@@ -139,34 +139,18 @@ <h2>DAGs</h2>
...
-
-                <!-- Retries -->
-                <a href="{{ url_for('airflow.tries', dag_id=dag.dag_id) }}">
-                    <span class="glyphicon glyphicon-duplicate" aria-hidden="true" data-original-title="Task Tries"></span>
+                  <span class="glyphicon glyphicon-certificate" aria-hidden="true" data-original-title="Graph View"></span>
                 </a>
 
-                <!-- Landing Times -->
-                <a href="{{ url_for("airflow.landing_times", dag_id=dag.dag_id) }}">
-                    <span class="glyphicon glyphicon-plane" aria-hidden="true" data-original-title="Landing Times"></span>
```
Comment (PrabodhaSrimal) [inline airflow/www/templates/airflow/dags.html]: Remove Gantt View too

File: airflow/www/views.py [hunk 4]
```diff
@@ -898,6 +898,58 @@ def run(self):
...
+    def trigger_with_conf(self):
+        dag_id = request.form['dag_id']
+        origin = request.args.get('origin') or "/admin/"
+        dag = dagbag.get_dag(dag_id)
+
+        if not dag:
+            flash("Cannot find dag {}".format(dag_id))
+            return redirect(origin)
+
+        execution_date = datetime.utcnow()
```
Comment (PrabodhaSrimal) [inline airflow/www/views.py:945]: Are all times in UTC?

**Review by PrabodhaSrimal — APPROVED** [review]

File: airflow/www/templates/airflow/dag.html (hunk 2 above)
Comment (VladMihaiC) [reply airflow/www/templates/airflow/dag.html:88]: don't you think the gantt can be useful? I think it's an easy way of getting an idea of time per task or even time per pipeline.

File: airflow/www/templates/airflow/dag.html (hunk 1 above)
Comment (VladMihaiC) [reply airflow/www/templates/airflow/dag.html]: I think Task Duration can give interesting information, can we just leave it in there? or is it too much?

**Review by VladMihaiC — APPROVED** [review]

File: airflow/www/views.py (hunk 4 above)
Comment (draffelt) [reply airflow/www/views.py:945]: Yes

File: airflow/www/templates/airflow/dag.html (hunk 2 above)
Comment (draffelt) [reply airflow/www/templates/airflow/dag.html:88]: I think the gantt is a nice overview of task time so I've left it in, but we can always change it later.

**Comment by draffelt** [comment]
Hey Guys, I'm going to merge this into a branch called roames-master instead of their 1.9-stable branch. Obviously we can still tag this branch as 1.9.0.x. When 2.0 is released we can then just keep this branch and rebase.

**Pull request merged**